src/css/parse.js

```diff
@@ -1358,6 +1358,45 @@ function getFunction() {
  * @param {number} i Token's index number
  * @return {number} Length of the identifier
  */
+function checkIdentIncludingHacks(i) {
+  const start = i;
+
+  if (i >= tokensLength) return 0;
+
+  if (tokens[i].type === TokenType.Asterisk) i++;
```

Comment:
I made a new function, which is 99% the same as the `checkIdent` function but adds:
```
if (tokens[i].type === TokenType.Asterisk) i++;
if (tokens[i].type === TokenType.NumberSign) i++;
```